### ISSUE TITLE
Suggestion for windows

### ISSUE THREAD

**Issue by Majticz** [zhiyiYo/PyQt-Frameless-Window#75]
**Is your feature request related to a problem? Please describe.**
No, it's just a suggestion to improve this library on windows

**Describe the solution you'd like**
I would like to recommend changes to the titlebar buttons so they return the default windows return value (`HTCLOSE`, `HTMAXBUTTON`, `HTMINBUTTON`).
Also, if cursor is at the edge of the screen, the buttons become in normal style rather than hover style

**Describe alternatives you've considered**
Tried implementing it myself using the `nativeEvent` function but the windows titlebar was shown

**Additional context**

**Comment by zhiyiYo** [comment]
User may call `FramelessWindow.setTitleBar()` to use their own title, so I can't detect the MinButton、MaxButton and CloseButton in `nativeEvent`

**Comment by zhiyiYo** [comment]
You can use `setToolTip` to achieve similar effect

**Comment by Majticz** [comment]
the `setToolTip` is not exactly what I want, I was already able to achieve the effect I want but I wasn't able to make the app cross platform so I switched to this and [PyQt-Fluent-Widgets](https://github.com/zhiyiYo/PyQt-Fluent-Widgets).

![image](https://user-images.githubusercontent.com/102349655/230784296-42d12a02-c9c1-43d9-817c-f1b5f2c9a57d.png)
here is what I already achieved, but for some it can only run on windows 11.

Anyways, thank you.
